ga4gh/server/datarepo.py

```diff
@@ -723,15 +723,15 @@ def insertReference(self, reference):
         sql = """
             INSERT INTO Reference (
                 id, referenceSetId, name, length, isDerived, md5checksum,
-                ncbiTaxonId, sourceAccessions, sourceUri)
+                species, sourceAccessions, sourceUri)
             VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?);
         """
         cursor = self._dbConnection.cursor()
         cursor.execute(sql, (
             reference.getId(), reference.getParentContainer().getId(),
             reference.getLocalId(), reference.getLength(),
             reference.getIsDerived(), reference.getMd5Checksum(),
-            reference.getNcbiTaxonId(),
+            json.dumps(reference.getSpecies()),
```

Comment:
You provide the class as a second argument (`protocol.OntologyTerm`) when inserting a reference set. Any reason for the discrepancy? Does it produce a nice exception?